src/internal.c

```diff
@@ -10315,7 +10318,7 @@ static int BuildCertHashes(WOLFSSL* ssl, Hashes* hashes)
             #endif
         }
     }
-#if ! defined( NO_OLD_TLS )
+#if !defined(NO_OLD_TLS)
     else {
         BuildMD5_CertVerify(ssl, hashes->md5);
         BuildSHA_CertVerify(ssl, hashes->sha);
```

Comment:
In the WOLFSSL_ALLOW_TLS_SHA1 case with TLS 1.2  and NO_OLD_TLS shouldn't the SHA1 hash be calculated here?